### PR TITLE
Chore: Added release action and dev hosting target

### PR DESCRIPTION
**Pull request by lemohammed**
Merge and PR will trigger firebase hosting workflow to deploy to dev site : ess-schulichhub-dev.web.app

Release will trigger firbease host workflow to deploy to hub.essucalgary.com 

Update includes new GitHub Action

Changelog:

- included new targets in .firebaserc
- included new hosting target configs in firebase.json
- changed github workflows:

## On Merge
now deploys to dev site on channelId: live
## on PR
now deploys to preview site to dev site
## On Release
now delpoys to hub.essucalgary.com

### CODE REVIEW

File: .firebaserc [hunk 1]
```diff
@@ -1,5 +1,17 @@
 {
   "projects": {
     "default": "project-hub-116d7"
+  },
+   "targets": {
+    "driver-profiles-b579c": {
+      "hosting": {
+        "dev": [
```
Comment (RatikKapoor) [inline .firebaserc:8]: Smart :)

**Review by RatikKapoor — APPROVED** [review]

**Comment by RatikKapoor** [comment]
@lemohammed we should remove the auto-pretty action, it tends to break a lot

File: .firebaserc (hunk 1 above)
Comment (lemohammed) [reply .firebaserc:8]: Thankss!

**Comment by lemohammed** [comment]
> @lemohammed we should remove the auto-pretty action, it tends to break a lot

Done

**Review by RatikKapoor — APPROVED** [review]

**Comment by lemohammed** [comment]
Lol i did dev and live as the wrong sites 😆 🤣

**Review by rjb75 — APPROVED** [review]

**Review by RatikKapoor — APPROVED** [review]

**Review by rjb75 — APPROVED** [review]

**Pull request merged**